scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/model/JobsPaginationModel.java

```diff
@@ -105,6 +112,41 @@ public void setFetchData(String startCursor, String endCursor, boolean first) {
         }
     }
 
+    /**
+     * Gets the number of the last page.
+     * @return the number of the last page.
+     */
+    public int getMaxPage() {
+        return maxPage;
+    }
+
+    /**
+     * Sets the total number of items without pagination.
+     * @param totalItems the total number of items without pagination.
+     */
+    public void setTotalItems(long totalItems) {
+        this.totalItems = totalItems;
+        int pageSize = this.getPageSize();
+        this.maxPage = ((int) this.totalItems / pageSize) - 1;
+        if (this.totalItems % pageSize != 0) {
+            this.maxPage++;
+        }
+        doActionOnListeners(listener -> listener.totalItemChanged());
+    }
+
+    public void setPage(int page) {
+        this.currentPage = page;
+        doActionOnListeners(listener -> listener.pageChanged());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)